src/components/ErrorBanner.tsx

```diff
@@ -4,21 +4,20 @@ import styled from 'styled-components';
 import { Typography } from '@material-ui/core';
 import { useSelector } from 'react-redux';
 import { RootState } from '../state/rootReducer';
+import { Alert, AlertTitle } from '@material-ui/lab';
 
-const StyledErrorDiv = styled.div`
-  padding: 0.5rem;
-  background-color: ${({ theme }) => theme.palette.danger.light};
+const StyledAlert = styled(Alert)`
   margin-top: 0.5rem;
   margin-bottom: 0.5rem;
-  text-align: center;
 `;
 
 interface ErrorBannerProps {
   userNeedsToBeLoggedIn?: boolean;
   error?: Error;
+  customErrorMessage?: string;
```

Comment:
jeg har lagt inn muligheten til å legge inn egne tekster istedenfor generiske "ups", men er det viktig å bruke tid på det ? Eksempelvis "Could not retrieve tags from server"

Reply:
Supert